examples/demo/src/products/GridList.tsx

```diff
@@ -106,7 +128,7 @@ const LoadedGridList = ({ ids, data, basePath, width }) => {
     );
 };
 
-const GridList = ({ loaded, ...props }) =>
+const GridList: FC<Props> = ({ loaded, ...props }) =>
     loaded ? <LoadedGridList {...props} /> : <LoadingGridList {...props} />;
 
-export default withWidth()(GridList);
+export default withWidth()(GridList as any);
```

Comment:
Not good. If you have to use any, try to look for an alternative solution. For instance, not using `withWidth`, but the new hooks provided by material-ui.

Reply:
Done